src/olympia/stats/templates/stats/reports/campaigns.html

```diff
@@ -1,3 +1,11 @@
 {% extends 'stats/report.html' %}
 
 {% set title = _('Download campaigns by Date') %}
+
+{% block stats_note_link %}
```

Comment:
As this is added identically to a few templates, would it be worth having a jinja/django (whichever these use) helper that outputs all/most of this?

Reply:
I thought about this too but it's kinda trivial so I dunno..